lib/tasks/vacations.rake

```diff
@@ -7,4 +7,26 @@ namespace :vacations do
       vacation.save(validate: false)
     end
   end
+
+  desc "Send a e-mail as remember with the pending vacations to be approve"
+  task alert_pending_approve: :environment do
+    Rails.logger.info "anything"
+    vacations = Vacation.pending
+
+    hr_pending_approves = vacations.where(hr_approver_id: nil)
+    if hr_pending_approves.any?
+      Rails.logger.info "Remembering the HR of pending vacations"
+      User.by_roles_in([:hr]).each do |hr_user|
+        VacationMailer.notify_pending_vacations(hr_user, hr_pending_approves)
+      end
+    end
+
+    commercial_pending_approves = vacations.where(commercial_approver_id: nil)
+    if commercial_pending_approves.any?
+      Rails.logger.info "Remembering the commercial of pending vacations"
+      User.by_roles_in([:commercial]).each do |commercial_user|
+        VacationMailer.notify_pending_vacations(commercial_user, commercial_pending_approves)
+      end
+    end
+  end
```

Comment:
Since you are querying for each one, why not create a scope like `Vacation.prending_approval_of(:hr)`